pkg/actions/db_service.go

```diff
@@ -8,17 +8,43 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"io"
 	"sync"
 	"time"
 
+	"github.com/georgysavva/scany/pgxscan"
 	"github.com/hashicorp/go-multierror"
+	"github.com/jackc/pgx/v4/pgxpool"
 	gonanoid "github.com/matoous/go-nanoid/v2"
+	"github.com/rs/xid"
 	"github.com/treeverse/lakefs/pkg/db"
 	"github.com/treeverse/lakefs/pkg/graveler"
+	"github.com/treeverse/lakefs/pkg/kv"
+	kvpg "github.com/treeverse/lakefs/pkg/kv/postgres"
 	"github.com/treeverse/lakefs/pkg/logging"
 	"github.com/treeverse/lakefs/pkg/stats"
+	"github.com/treeverse/lakefs/pkg/version"
+	"google.golang.org/protobuf/proto"
 )
 
+//nolint:gochecknoinits
+func init() {
+	kvpg.RegisterMigrate(packageName, Migrate, []string{"actions_run_hooks", "actions_runs"})
+}
+
+const (
+	runWorkers       = 10
+	taskWorkers      = 10
+	migrateQueueSize = 100
+)
```

Comment:
Question - any specific reason to use these numbers?

Reply:
No specific reason - mainly trial and error. I noticed that more than 10 workers do not really make any significant performance change and the decision for queue size was to make it at least an order of magnitude bigger than num of workers but not too big